src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -152,15 +152,24 @@ export const ConfirmSummary = ({
   ])
 
   const quoteHasError = useMemo(() => {
+    const tradeQuoteError = activeQuoteErrors?.[0]
+
+    // Ensures final trade quote errors are not displayed at input time for one or two render cycles as tradeQuoteSlice when reset
+    // if backing out from an errored final quote back to input
+
+    if (tradeQuoteError && tradeQuoteError.error === TradeQuoteError.FinalQuoteMaxSlippageExceeded)
+      return false
+    if (tradeQuoteError && tradeQuoteError.error === TradeQuoteError.FinalQuoteMaxSlippageExceeded)
```

Comment:
tsc was somehow sad when I was doing `.includes()` checks instead of two explicit lines